master/src/main/scala/org/apache/celeborn/service/deploy/master/http/api/v1/WorkerResource.scala

```diff
@@ -73,10 +73,15 @@ class WorkerResource extends ApiRequestContext {
   @POST
   @Path("/exclude")
   def excludeWorker(request: ExcludeWorkerRequest): HandleResponse = {
-    val (success, msg) = httpService.exclude(
-      request.getAdd.asScala.map(ApiUtils.toWorkerInfo).toSeq,
-      request.getRemove.asScala.map(ApiUtils.toWorkerInfo).toSeq)
-    new HandleResponse().success(success).message(msg)
+    if (master.isMasterActive == 1) {
+      val (success, msg) = httpService.exclude(
+        request.getAdd.asScala.map(ApiUtils.toWorkerInfo).toSeq,
+        request.getRemove.asScala.map(ApiUtils.toWorkerInfo).toSeq)
+      new HandleResponse().success(success).message(msg)
+    } else {
+      throw new BadRequestException(
```

Comment:
IMO, the users should be told which master node is leader.

Reply:
updated